app/scripts/components/prototypo-panel.components.jsx

```diff
@@ -67,7 +67,13 @@ export default class PrototypoPanel extends React.Component {
 	}
 
 	toggleView(name) {
-		const newViewMode = _.intersection(_.xor(this.state.uiMode, [name]), this.availableMode);
+		// if we are toggling glyph mode, we want glyph list to be hidden
+		const modes = (
+			name === 'glyph'
+				? _.without(this.state.uiMode, 'list')
+				: this.state.uiMode
+		);
+		const newViewMode = _.intersection(_.xor(modes, [name]), this.availableMode);
```

Comment:
If available mode has all the possible mode it is useless to do intersection. Remove this.available and the intersection.